test/k6/src/tests/instances.js

```diff
@@ -276,15 +280,83 @@ function TC10_SoftDeleteInstance(data) {
   addErrorCount(success);
 }
 
-function TC11_HardDeleteInstance(data) {
+//TC12 - Get all instances for party looked up with a person number
+function TC11_GetInstances_ByPersonNumber(data) {
+  var instanceOwnerIdentifier = "Person:" + data.personNumber;
+  var filters = {
+    "appId": data.org + "/" + data.app,
+  };
+  var options = {
+    "instanceOwnerIdentifier" : instanceOwnerIdentifier,
+  };
+  // var res = instancesApi.getInstances(data.userToken, filters, options);
+  var res = instancesApi.getInstances(data.userToken, filters, options);
+
+  var success = check(res, {
+    "TC11_GetInstances_ByPersonNumber: Get instance for party. Status is 200": (
+      r
+    ) => r.status === 200,
+  });
+  addErrorCount(success);
+
+  const firstInstance = JSON.parse(res.body).instances[0];
+  if (firstInstance) { // Making sure there actually exists instances for the instanceOwner
+    const instanceIdSplit = firstInstance["id"].split("/");
+
+    success = check(res, {
+      "TC11_GetInstances_ByPersonNumber: Get instance for party. InstanceId has expected format":
+        instanceIdSplit.length === 2,
+    });
+    addErrorCount(success);
+  }
+}
+
+//TC13 - Get all instances for party looked up with an organisation number
+function TC12_GetInstances_ByOrgNumber(data) {
+  var instanceOwnerIdentifier = "Organisation:" + data.orgNumber;
```

Comment:
are we ensuring before this tests runs that there actually exists any instances for the given orgnumber?